mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxConstants.java

```diff
@@ -46,4 +46,6 @@
     /** The maximum number of annotations on a mailbox */
     int DEFAULT_LIMIT_ANNOTATIONS_ON_MAILBOX = 10;
 
+    /** The limitation of mailbox's name */
+    int DEFAULT_LIMIT_MAILBOX_NAME_SIZE = 256;
```

Comment:
The idea is making it can configurable in the configuration file (Eg: imapserver.xml or the other files) but as I tried with mailbox annotation, I haven't known how to load those value. So I make it as a constant in the MailboxConstants file. Then 256 is just my assumption value.
Could you guide me how to make it configurable?